etc/scripts/ruby/disasm.rb

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021, Compiler Explorer Authors
```

Comment:
I wonder if https://github.com/compiler-explorer/tools is a better place for this. I'll ask

Reply:
I wonder no more. We already use a python script for a similar pourpose, so I guess this is ok! :)